common/presenters/single-requests-to-table-component.js

```diff
@@ -7,83 +7,95 @@ const componentService = require('../services/component')
 const moveToCardComponent = require('./move-to-card-component')
 const tablePresenters = require('./table')
 
-function singleRequestsToTable(moves) {
-  const tableConfig = [
-    {
-      head: {
-        text: i18n.t('name'),
-        attributes: {
-          width: '220',
+function singleRequestsToTable({ isSortable = true, query } = {}) {
```

Comment:
It might be good to add some tests around calling this method with the different properties.

It currently looks like the tests only test the default values for these options.

Reply:
Tests have been added for the use of `isSortable` and `query`